### ISSUE TITLE
Wire up segment analytics

### ISSUE THREAD

**Issue by StephenHaney** [modulz/modulz-website#1]
Once we have some structure in place I can help show how to wire up our segment integration.

**Comment by peduarte** [comment]
@StephenHaney Hey Stephen, I'm ready to start on this, but have a few questions.

I've just logged in to our Segment dashboard, and noticed the same tracking ID is being used for the current Marketing Website, as well as the [App itself](https://github.com/modulz/modulz/blob/develop/packages/core/public/analytics.js#L50).

Do you want to continue using the same tracking ID for everything?

Cheers!

**Comment by StephenHaney** [comment]
We should probably switch to using a Modulz GA account so it's managed through our GSuite like everything else... I set up a new one here:

Here's the new GA tracking ID, we'll need to switch Segment to use this ID:
```
UA-160992000-1
``

**Comment by StephenHaney** [comment]
Do you think we should separate out the App from the Marketing site in analytics? Might be a good idea.

If so, we just need to create a new "Property" in that new GA account for the App, and then set up separate Sources and Destinations in Segment. Let me know if you want any help with this or another set of eyes.

You should have full permissions in the GA account and Segment.

**Comment by peduarte** [comment]
@StephenHaney That's brilliant, thanks for this 🙌 

Im not an analytics expert by far, so would love a second pair of eyes, here's what I've done:

- Google Analytics now contains 2 properties, as you suggested
  - Marketing `UA-160992000-1`
  - App `UA-160992000-2`
- Segment now contains 2 sources, they've been connected with their respective GA ID
  - Marketing Prod
  - App Prod
- In Segment, the App Prod source, is the same as the existing source that was there before, I just renamed it, this means that any segment tracking work that was done in the Core App can remain. If you want to start fresh, let me know and I can delete this source and recreate it

**Comment by StephenHaney** [comment]
Sounds great to me!

I think we can continue using the same source ID as before the App.

I noticed in Segment, the "App Prod" destination was pointing at the `-1` tracking code, so I changed it to `-2`. Tested this using the GA real time view and it looks like it's coming through to the right GA property from the app.

The App is currently sending analytics from all environments (not just prod) but I'll fix that separately.
